dotcom-rendering/package.json

```diff
@@ -28,6 +28,7 @@
     "cypress:run": "cypress run",
     "cypress:open:e2e": "cypress open --e2e --browser=chrome",
     "cypress:run:e2e": "cypress run --spec 'cypress/e2e/**/*'",
+    "cypress:run:percy": "percy exec -- cypress run --config specPattern='cypress/snapshot/**/*'",
```

Comment:
This overrides the default `specPattern` i.e. `e2e/**/*.cy.*` so we can have tests outside of the default convention.

Otherwise tests would have to be in `e2e/` and an `excludeSpecPattern ` specified for snapshots when running regular e2e tests.

Reply:
So for clearer separation we now have:

```
/cypress
  /e2e
  /snapshot
```